.github/labeler.yml

```
@@ -1,13 +1,12 @@
 Documentation:
-- doc/source/**/*
+- changed-files:
+  - any-glob-to-any-file: ['doc/source/**/*']
 Maintenance:
-- .github/**/*
-- codecov.yml
-- .flake8
-- .coveragerc
-- doc/styles/**/*
-- pyproject.toml
+- changed-files:
+  - any-glob-to-any-file: ['.github/**/*', 'codecov.yml', '.flake8', '.coveragerc', 'doc/styles/**/*', 'pyproject.toml']
```

You can also have the files like this if you prefer it

```suggestion
  - any-glob-to-any-file: 
    -'.github/**/*'
    -'codecov.yml'
    
    ...
```

---

Interesting, I thought it was not possible. I will implement it this way. Thank you for your feedback!